include/iso15118/session/feedback.hpp

```diff
@@ -33,11 +35,26 @@ struct DcMaximumLimits {
     float power{-1};
 };
 
+struct DisplayParameters {
+    std::optional<float> present_soc;
+    std::optional<float> minimum_soc;
+    std::optional<float> target_soc;
+    std::optional<float> maximum_soc;
+    std::optional<float> remaining_time_to_minimum_soc;
+    std::optional<float> remaining_time_to_target_soc;
+    std::optional<float> remaining_time_to_maximum_soc;
+    std::optional<float> battery_energy_capacity;
+    std::optional<bool> inlet_hot;
+};
+
 struct Callbacks {
     std::function<void(Signal)> signal;
     std::function<void(const DcChargeTarget&)> dc_charge_target;
     std::function<void(const DcMaximumLimits&)> dc_max_limits;
     std::function<void(const message_20::Type&)> v2g_message;
+    std::function<void(const std::string&)> evccid;
+    std::function<void(const std::string&)> selected_protocol;
+    std::function<void(const DisplayParameters&)> display_parameters;
```

Comment:
If this is getting larger, it would probably make sense to transmit a variant via a single function.

Reply:
That's right. If I expand this again in the future, I'll change it to a std::variant.